terraform/18f.gov.tf

```diff
@@ -1545,36 +1512,30 @@ resource "aws_route53_record" "d_18f_gov__acme-challenge_federalist-proxysite-te
   records = ["qiz18V0W9mHROFHTHqE4_Bn8NIByqMhoLvkKQ3h4Zxg"]
 }
 
-resource "aws_route53_record" "d_18f_gov_ux-guide_18f_gov_a" {
+resource "aws_route53_record" "d_18f_gov__acme-challenge_ux-guide_18f_gov_txt" {
   zone_id = aws_route53_zone.d_18f_gov_zone.zone_id
-  name    = "ux-guide.18f.gov."
-  type    = "A"
-
-  alias {
-    name                   = "d2mhwjcivqpysk.cloudfront.net."
-    zone_id                = local.cloud_gov_cloudfront_zone_id
-    evaluate_target_health = false
-  }
+  name    = "_acme-challenge.ux-guide.18f.gov."
+  type    = "TXT"
```

Comment:
Is there a reason this is a TXT and not a CNAME?

Reply:
There were a few TXT records already present in the file (there's one for the engineering guide as well). Should I be removing the TXT records for the guides?

Reply:
There should be no TXT records for `_acme-challenge.ux-guide.18f.gov` subdomain.